Kickstarter-iOS/Views/Controllers/CreatePasswordViewController.swift

```diff
@@ -93,8 +187,57 @@ final class CreatePasswordViewController: UITableViewController {
     guard let footerView = tableView.dequeueReusableHeaderFooterView(withClass: className)
       as? SettingsGroupedFooterView else { return nil }
 
-    footerView.configure(with: Strings.Password_min_length_message())
+    footerView.configure(with: self.viewModel.outputs.currentValidationLabelText())
 
     return footerView
   }
+
+  // swiftlint:disable line_length
```

Comment:
Not a huge fan of wrapping `UITableView` + `Delegate` or `DataSource` methods on multiple lines so I've disabled Swiftlint...happy to change this if it's not ideal